<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rentacar</class>
 <widget class="QDialog" name="rentacar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>둘이옵서예 렌터카 예약</string>
  </property>
  <widget class="QTableWidget" name="carlist">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>516</width>
     <height>351</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>65</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>차종</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>종류</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>제원</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>렌트비용</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>남은수량</string>
    </property>
   </column>
  </widget>
  <widget class="QRadioButton" name="all_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>전체</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="gasoline_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>휘발유</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="desel_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>경유</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="gas_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>LPG</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="battery_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>전기차</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inquiry_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>400</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>렌터카 조회</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reservation_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>실시간 예약</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
